Assets/Scripts/EOSPlayerDataStorageManager.cs

```diff
@@ -483,5 +483,11 @@ private void OnFileRemoved(ref DeleteFileCallbackInfo data)
 
             QueryFileList();
         }
+
+        public override void RemoveCallbacks()
+        {
+            base.RemoveCallbacks();
+            OnFileListUpdated = null;
+        }
```

Comment:
A few things with respect to this:
- It does not appear that the `RemoveCallbacks` function (either the base implementation or the overridden one) is ever called.
- In general, `Action` and `event` field members do not need to be set to null - because the relationship between them and the "clients" who subscribe to them is such that no error should occur if the action goes away - the client just doesn't get called. The client doesn't store a reference to the callback that is at risk of causing a problem.

For these reasons, I believe that this function, and the function that it overrides can be removed.

Reply:
Oh! That's a relief, okay. I had always thought Actions needed to be cleaned up in this way.

These are supposed to be called during the teardown of the test. Looks like it is there. Will remove that part of the teardown, and remove this function.